backend/src/main/java/org/uresti/pozarreal/service/impl/HousesServiceImpl.java

```diff
@@ -134,4 +132,29 @@ public HouseByUser saveHouseByUser(HouseByUser houseByUser) {
 
         return HousesByUserMapper.entityToDto(housesByUserRepository.save(HousesByUserMapper.dtoToEntity(houseByUser)));
     }
+
+    @Override
+    public ArrayList<PaymentByConcept> getPaymentsHouse(String houseId) {
+        LocalDate startOfYear = LocalDate.now().withYear(LocalDate.now().getYear()).withDayOfYear(1);
```

Comment:
Are you sure ".withYear(LocalDate.now().getYear())" is doing something? .. I mean, the base date is LocalDate.now()

Reply:
i tried to get current date and end with one year more